docs/source/getting-started.md

```diff
@@ -44,6 +44,16 @@ iwr 'https://rover.apollo.dev/win/v0.17.2' | iex
 
 Rover is distributed on npm for integration with your JavaScript projects.
 
+#### Installation from NPM mirror
```

Comment:
I'm not sure I follow. The docs here say this feature is about an NPM mirror, but isn't the change about making a mirror of Apollo's bespoke rover.apollo.dev tarball download site, not of NPM at all?

Reply:
Yeah, you're right.

I conflated the typical use case of "if you have an npm mirror, you're going to need a mirror for the binary".  I _think_ this is the language other binary wrapper/ABI packages use as well though, so might be ok if it's what users are used to searching for.

Reply:
We ended up addressing this in https://github.com/apollographql/rover/pull/1696
Does that look right to you?